pyproject.toml

```diff
@@ -128,7 +128,11 @@ addopts = [
 ]
 
 [tool.coverage.report]
-omit = [".env/*"]
+omit = [
+  ".env/*",
+  "project_euler/*",
+  "scripts/*"
```

Comment:
Let's leave scripts in.  I dislike not testing things.
```suggestion
```

I think the coverage report is way too long and users need to scroll across hundreds of lines (that no one reads) to find their failing pytests.

Would it be possible to put coverage in a separate GitHub Actions step so its output is not tagged directly on the end of pytest output?

Reply:
We could break up `pytest` into two separate workflows, one for tests and one for coverage.

Current:
```yml
- name: Run tests
        # TODO: #8818 Re-enable quantum tests
        run: pytest
            --ignore=quantum/q_fourier_transform.py
            --ignore=project_euler/
            --ignore=scripts/validate_solutions.py
            --cov-report=term-missing:skip-covered
            --cov=. .
```

Proposed:
```yml
- name: Run tests
        # TODO: #8818 Re-enable quantum tests
        run: pytest
            --ignore=quantum/q_fourier_transform.py
            --ignore=project_euler/
            --ignore=scripts/validate_solutions.py
- name: Check test coverage
        run: pytest
            --cov-report=term-missing:skip-covered
            --cov=.
```